dpbench/benchmarks/rambo/rambo_numba_dpex_k.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import math
+
+import numba_dpex
+import numpy
+from numba_dpex import DEFAULT_LOCAL_SIZE, kernel
+
+
+def gen_rand_data(nevts, nout):
```

Comment:
Why is this initialization repeated in each implementation? The logic to initialize `C1`,`F1`,`Q1` should be moved into the `initialize` function and the `initialize` function should return these matrices.
The `output_args` of the `init` section of the JSON should add these arrays as outputs from `initialize`. Then these arrays should become `input_args` for the benchmark. 

The way it is written right now the data copy will always happen inside a dpex_k as we are always going to pass in NumPy arrays.